installers/agent/win/config/wrapper-agent.conf

```diff
@@ -62,8 +62,8 @@ set.AGENT_STARTUP_ARGS=-Xms128m -Xmx256m %JVM_DEBUG% %GC_LOG% "-DJAVA_SYS_MON_TE
 
 # Application parameters.  Add parameters as needed starting from 1
 wrapper.app.parameter.1=com.simontuffs.onejar.Boot
-wrapper.app.parameter.2=%GO_SERVER%
-wrapper.app.parameter.3=8153
+wrapper.app.parameter.2=-serverUrl
```

Comment:
If users have either of these two properties overridden in their `wrapper-properties.conf`, the *new* values from wrapper-agent.conf will be ignored. Meaning, there could be users running Go server on port 8000, so they would have an entry `wrapper.app.parameter.3=8000` in their wrapper-properties.conf. Upon upgrade, bootstrapper expects `GO_SERVER_URL` to be passed along but now we would get a port number instead. Either we should handle it (meaning log proper error if this happens), or have some sort of warning somewhere (in docs/release notes?) stating the same.